test/toolTests.ts

```diff
@@ -1,20 +1,19 @@
-/// <reference path="../typings/index.d.ts" />
-/// <reference path="../_build/tool.d.ts" />
-
 import assert = require('assert');
 import path = require('path');
 import fs = require('fs');
 import shell = require('shelljs');
 import os = require('os');
+
+import * as mocha from 'mocha';
```

Comment:
I tried those two and a few other permutations and couldn't get them working. I also searched around and couldn't find examples of how to do it. I am new to ts but it's strange to me that there isn't a way to do this :)

I added the newline to separate the two import types, I believe the second set is the ES6 way. I can get rid of it if that's the format we normally follow.